sorts/bubble_sort.py

```diff
@@ -22,14 +22,15 @@ def bubble_sort(collection):
     True
     """
     length = len(collection)
-    for i in range(length-1):
-        swapped = False
+    swapped = True
+    i = 0
+    while swapped == True and i < n: # Stop iteration if the collection is sorted.
```

Comment:
What is `n`? Do you mean `length`?

Reply:
https://travis-ci.org/TheAlgorithms/Python/builds/586149305#L600

Reply:
Thanks. Replaced n by length